templates/default/config.ini.erb

```diff
@@ -25,14 +26,16 @@ hostname = <%= @hostname %>
 group-name = <%= node[:kagent][:group_name] %>
 hadoop-home = <%= @hops_dir %>/hadoop
 certs-dir = <%= node[:kagent][:certs_dir] %>
+certs-user = <%= node['kagent']['certs_user'] %>
```

Comment:
Can you also fix this test https://github.com/logicalclocks/kagent-chef/blob/master/files/default/kagent_utils/tests/test_kagent_config.py and run `pytest` ?